app/enterprise/1.5.x/kong-for-kubernetes/install.md

```diff
@@ -158,13 +152,223 @@ $ export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip
+
+{% navtabs %}
+{% navtab kubectl %}
+1. Set up the credentials:
+    ```
+    $ kubectl create secret -n kong docker-registry kong-enterprise-edition-docker \
+        --docker-server=kong-docker-kong-enterprise-edition-docker.bintray.io \
+        --docker-username=<your-bintray-username> \
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    kubectl create secret generic kong-enterprise-superuser-password  -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% navtab OpenShift oc %}
+1. Set up the credentials:
+    ```
+    $ oc create secret -n kong docker-registry kong-enterprise-edition-docker \
+        --docker-server=kong-docker-kong-enterprise-edition-docker.bintray.io \
+        --docker-username=<your-bintray-username> \
+        --docker-password=<your-bintray-api-key>
+    ```
+
+2. (Optional) Create a password for the super admin:
+    ```
+    oc create secret generic kong-enterprise-superuser-password -n kong --from-literal=password=<your-password>
+    ```
+    Though not required, this is recommended for users that wish to use RBAC, as it cannot be done after initial setup.
+{% endnavtab %}
+{% endnavtabs %}
+### Enable the sessions plugin for Manager and Dev Portal
+In the following steps, replace `<your-password>` with a secure password.
+
+{% navtabs %}
+{% navtab kubectl %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ cat > admin_gui_session_conf
+    {"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ cat > portal_session_conf
+    {"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+
+3. Create secret:
+    ```
+    $ kubectl create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
+    ```
+{% endnavtab %}
+{% navtab OpenShift oc %}
+
+1. Create a sessions config file for Kong Manager:
+    ```
+    $ cat > admin_gui_session_conf
+    {"cookie_name":"admin_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+2. Create a sessions config file for Kong Dev Portal:
+    ```
+    $ cat > portal_session_conf
+    {"cookie_name":"portal_session","cookie_samesite":"off","secret":"<your-password>","cookie_secure":false,"storage":"kong"}
+    ```
+
+3. Create secret:
+    ```
+    $ oc create secret generic kong-session-config -n kong --from-file=admin_gui_session_conf --from-file=portal_session_conf
+    ```
+{% endnavtab %}
+{% endnavtabs %}
+### Prepare the values.yaml
```

Comment:
@hbagdi is there any way we can add a sample `values.yaml` specific to this package to the kong charts repo? That would make this section far easier to follow.

Reply:
ping @rainest

Reply:
We plan to add these for a variety of scenarios: https://trello.com/c/t9sSfTi4/378-provide-example-valuesyamls-for-common-configurations

@lena-larionova do you have examples ready to go? The only things I'd want for a PR is that everything is placed under an `example-values` directory (doesn't exist yet) and that it either installs successfully or installs successfully if users create all referenced secrets.

Reply:
Assuming probably no given the current format in this document, and we needed to write some anyway. https://github.com/Kong/charts/tree/feat/example-values with the caveat that this is still a draft and I haven't confirmed that these do in fact work.

Reply:
@rainest Harry gave me very bare bones example that I used for testing this out - I tweaked it to make it work with this install topic, I can send you that version.

Reply:
https://github.com/Kong/charts/pull/134 has the same, now properly tested.